src/stylelint/rules/use-defined-css-var-when-available/index.js

```diff
@@ -0,0 +1,81 @@
+const stylelint = require("stylelint");
+const valueParser = require("postcss-value-parser");
+const { getPropsToAllowedCssVars } = require("./parse-monday-css");
+
+const { report, ruleMessages, validateOptions } = stylelint.utils;
+
+const ruleName = "monday-ui-style/use-defined-css-var-when-available";
+const messages = ruleMessages(ruleName, {
+  expected: (original, expected) => {
+    const asArray = Array.isArray(expected) ? expected : [expected];
+    const multipleValues = asArray.length > 1;
+    const expectedMsg = multipleValues ? `one of vars: \n${asArray.join("\n")}\n` : `"var(${asArray[0]})"`;
+    return `Expected "${original}" to be ${expectedMsg}`;
```

Comment:
[PETTY]

```suggestion
    return `Expected "${original}" to be "${expectedMsg}"`;
```

Reply:
the `expectedMsg` var is adding the wrapping `"` 😄 
see for example:
![image](https://user-images.githubusercontent.com/96776835/162151798-9eb7abf9-366b-467b-b1a3-5ab294cd2892.png)

When there are multiple replacements I didn't add the wrapping `"var(...` since it added a lot of noise. I only print the var names in this case
![image](https://user-images.githubusercontent.com/96776835/162151979-3cc46a12-7617-4299-91c2-c2d7a63e1907.png)

Do you think I should change it?

Reply:
Oh my bad, I missed it 😅